### DISCUSSION TITLE
Transfer SID disabled rules from Security Onion 1 to Security Onion 2

### DISCUSSION THREAD

**Discussion by wompRS** [Security-Onion-Solutions/securityonion#5397] [General] [answered]
I am currently migrating our SECO solution from seco1 to seco2 (Security onion 1, 2). The way of disabling rules has changed obviously and we had done a lot of tuning on our system. What would be the best way of exporting the rules from seco1 and importing into seco2?

**Comment by dougburks** [comment, marked as answer]
In Security Onion 16.04, you would have disabled rules in `/etc/nsm/pulledpork/disablesid.conf`:
https://docs.securityonion.net/en/16.04/alerts.html#disable-the-sid

In Security Onion 2, you can disable rules via the `so-rule` command or in the minion pillar file:
https://docs.securityonion.net/en/2.3/managing-alerts.html#disable-the-sid
